tests/test_metrics_unit.py

```
@@ -498,14 +526,130 @@ async def test_maybe_checkpoint_failure(self, mock_consumer):
+
+        # Temporarily make <collector> the global so default __init__ picks it up.
+        previous = _get()
+        set_metrics_collector(collector)
+        try:
+            cp = MemoryCheckPointer("test")
+            await cp.allocate("shard-0")
+            await cp.checkpoint("shard-0", "seq-1")
+        finally:
+            set_metrics_collector(previous)
+
+        key = f"consumer_checkpoint_success_total{{shard_id=shard-0,stream_name=<standalone>}}"
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)